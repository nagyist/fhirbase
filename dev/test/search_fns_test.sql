--db:fhirb
--{{{

SELECT build_sorting('Patient', '_sort=given');

--}}}
