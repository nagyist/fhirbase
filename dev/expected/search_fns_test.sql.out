SELECT build_sorting('Patient', '_sort=given');
                             build_sorting                             
-----------------------------------------------------------------------
                                                                      +
  ORDER BY (json_get_in(patient.content, '{name,given}'))[1]::text ASC
(1 row)

